src/phoenix/server/api/routers/v1/experiments.py

```diff
@@ -85,17 +87,39 @@ async def create_experiment(request: Request) -> Response:
 
         # generate a semi-unique name for the experiment
         experiment_name = payload.get("name") or _generate_experiment_name(dataset_name)
+        project_name = f"Experiment-{getrandbits(96).to_bytes(12, 'big').hex()}"
```

Comment:
It could be helpful to include the dataset name, e.g., `optimize-prompt-template Experiment-<hash>`.

Reply:
punting on this, since it can be changed later